Makefile

```diff
@@ -91,7 +98,7 @@ lint: install-golangci-lint
 tidy: ## add missing and remove unused modules
 	 go mod tidy
 
-format: ## run go & rust formatters
+format: check-rust ## run go & rust formatters
```

Comment:
it's not critical for format target, maybe remove `check-rust` from here?

Reply:
doesn't it need to have rust to format?  From what I saw it formats both go and rust

Reply:
it does (but in theory having `rustfmt` doesn't mean that you need `rustc`). But my point is that we don't need specific rust version in order to format rust codebase.

Rust from version to version doesn't change language significantly, they use so-called editions to introduce breaking changes into the language. And editions are controlled by cargo.toml